lib/Jsrt/Jsrt.cpp

```diff
@@ -2221,6 +2221,36 @@ CHAKRA_API JsGetIndexedPropertiesExternalData(
     END_JSRT_NO_EXCEPTION
 }
 
+CHAKRA_API JsLessThan(_In_ JsValueRef object1, _In_ JsValueRef object2, _Out_ bool *result)
+{
+    return ContextAPIWrapper<JSRT_MAYBE_TRUE>([&](Js::ScriptContext *scriptContext, TTDRecorder& _actionEntryPopper) -> JsErrorCode {
+        PERFORM_JSRT_TTD_RECORD_ACTION(scriptContext, RecordJsRTEquals, object1, object2, false);
```

Comment:
I'm pretty sure we will need a new TTD action for recording these. Right now we're recording doing a non-strict-equals with `RecordJsRTEquals` and the flag value `false` to indicate not `is-strict`. We probably want to add a new action like `RecordJsRTLessThan` with a flag to say if it should be `or-equal`.